src/internal/tester/its/resourceService.go

```diff
@@ -0,0 +1,25 @@
+package its
+
+import "github.com/alcionai/corso/src/pkg/path"
+
+type ResourceServicer interface {
+	Resource() IDs
+	Service() path.ServiceType
+}
+
+type resourceAndService struct {
```

Comment:
could you please add an assignment that ensures continued interface compliance for this struct?
```suggestion
var _ ResourceServicer = resourceAndService{}

type resourceAndService struct {
```

Reply:
good catch, yes!